elastic/kibana
# [Synthetics] Overview - standardize queries for monitor duration metric item

## Summary

Resolves https://github.com/elastic/kibana/issues/145922
Resolves https://github.com/elastic/kibana/issues/145270

Fixes a regression from https://github.com/elastic/kibana/issues/143309

Standardizes the way the monitor duration metric is queried, ensuring that both monitor types, UI and Project, are queries in the same way.

The changes were also carried into changes to the Monitor Flyout

Before (duration for UI monitor shows, but not the duration for project monitors)
![image](https://user-images.githubusercontent.com/11356435/203143905-4564b14e-a2af-402c-a2c1-5b95d22ad1e3.png)

After (duration shows for both monitor types)
<img width="1366" alt="Screen Shot 2022-11-21 at 2 39 09 PM" src="https://user-images.githubusercontent.com/11356435/203144056-d74617e5-6745-4c17-9a68-e5164f4be689.png">

### Testing
1. Create at least one project monitor and one UI monitor
2. Ensure that both monitor types display the duration metric on the Overview page after running
3. In the actions popover, click edit monitor. Ensure the page redirects to the monitor edit page appropriately
4. In the actions popover, click go to monitor. Ensure that the page redirects to the monitor details page appropriately
5. In the actions popover, disable the monitor. Ensure it is successful.
6. In the actions popover, click inspect. Ensure the monitor flyout appears, then close the flyout.
7. Click on both monitor types, ensure the monitor flyout appears
8. Ensure the monitor duration metric appears in the monitor flyout
9. Ensure the last test run appears in the monitor flyout
10. Ensure both enable toggles work in the monitor flyout (in the body of the flyout and actions popover next to the flyout title)
11. Ensure the go to monitor link works in the monitor flyout actions popover next to the monitor title
12. Ensure the edit monitor link works in the monitor flyout actions popover next to the monitor title
